### PR TITLE
Audit adapter

### PR DESCRIPTION
**Pull request by banterCZ**
Issue #352

### CODE REVIEW

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationOtpService.java [hunk 1]
```diff
@@ -256,21 +256,24 @@ private void sendOtpCode(String processId, boolean isResend) throws OnboardingPr
...
-                    .locale(LocaleContextHolder.getLocale())
-                    .otpType(SendOtpCodeRequest.OtpType.USER_VERIFICATION)
-                    .build();
             onboardingProvider.sendOtpCode(request);
         } catch (OnboardingProviderException e) {
             logger.warn("OTP code delivery failed, error: {}", e.getMessage(), e);
             throw new OnboardingOtpDeliveryException(e);
         }
+
+        final String resentPrefix = isResend ? "Re-sent" : "Sent";
```
Comment (romanstrobl) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationOtpService.java:275]: I would prefer `Resent` over `Re-sent`, the first spelling is used frequently.

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java [hunk 2]
```diff
@@ -485,5 +490,7 @@ private void resendOtp(final OnboardingProcessEntity process, final String otpCo
             logger.warn("OTP code resend failed, error: {}", e.getMessage(), e);
             throw new OnboardingOtpDeliveryException(e);
         }
+
+        auditService.auditOnboardingProvider(process, "Re-sent activation OTP for user: {}", userId);
```
Comment (romanstrobl) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java:494]: I would prefer `Resent` over `Re-sent`, the first spelling is used frequently.

**Review by romanstrobl — APPROVED** [review]
I have one nit pick related to the `re-sent` word.

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java (hunk 2 above)
Comment (banterCZ) [reply enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java:494]: No problem, I will change it. I was not sure what to choose.

**Pull request merged**